app/controllers/demo/school_search_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+module Demo
+  class SchoolSearchController < ApplicationController
+    def index
```

Comment:
How does rubocop feel about this? It usually prefers empty methods on a single line

Reply:
burned it with rubocop